nbs/docs/getting-started/01_introduction.ipynb

```diff
@@ -102,45 +102,74 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "401a785d",
+   "id": "ea728145-d6cb-4eab-888a-6033b3388953",
    "metadata": {},
    "outputs": [],
    "source": [
-    "#| eval: false\n",
+    "import logging\n",
+    "\n",
     "import numpy as np\n",
```

Comment:
Non-blocking we could get rid of numpy, pandas  and IPython imports here too

Reply:
Removed